packages/interface-adapter/lib/web3-interface-adapter.ts

```diff
@@ -0,0 +1,12 @@
+import { Web3Shim, Web3ShimOptions } from "./web3-shim";
+
+export interface Web3InterfaceAdapterOptions extends Web3ShimOptions {}
+
+export class Web3InterfaceAdapter extends Web3Shim {
```

Comment:
Making it extend the shim? Is this necessary? Can you put the shim as an instance variable and do things like `this.web3.eth.net.getId()` instead?

Reply:
Well, looks like if we don't extend then none of the Web3/Shim methods are available to the Class. 😕